### PR TITLE
progress bar and screen out reporting simulation parameters etc.

### PR DESCRIPTION
**Pull request by joglekara**

### CODE REVIEW

File: run_vlapy.py [hunk 1]
```diff
@@ -20,6 +20,10 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+import os
+from datetime import datetime
+
```
Comment (StanczakDominik) [inline run_vlapy.py:25]:
```suggestion
import pprint
```

File: run_vlapy.py [hunk 2]
```diff
@@ -57,9 +61,25 @@
         )
     )
 
+    mlflow_exp_name = "Landau Damping-test"
+
+    print("Starting VlaPy at " + datetime.now().strftime("%m/%d/%Y %H:%M:%S"))
+    print("MLFlow experiment name: " + mlflow_exp_name)
+    print("mlruns folder located at " + os.getcwd())
+    print("Run parameters: ")
+    print(all_params_dict)
```
Comment (StanczakDominik) [inline run_vlapy.py:70]:
```suggestion
    pprint.pprint(all_params_dict)
```

File: run_vlapy.py [hunk 3]
```diff
@@ -57,9 +61,25 @@
...
+    print("Driver parameters: ")
+    print(pulse_dictionary)
```
Comment (StanczakDominik) [inline run_vlapy.py:72]:
```suggestion
    pprint.pprint(pulse_dictionary)
```

**Comment by joglekara** [comment]
thanks @StanczakDominik !

**Pull request merged**